packages/edit-widgets/src/blocks/widget-area/edit/index.js

```diff
@@ -27,14 +28,47 @@ export default function WidgetAreaEdit( {
 	);
 	const { setIsWidgetAreaOpen } = useDispatch( 'core/edit-widgets' );
 
+	const wrapper = useRef();
+	const setOpen = useCallback(
+		( openState ) => setIsWidgetAreaOpen( clientId, openState ),
+		[ clientId ]
+	);
+	const isDragging = useIsDragging();
+	const isDraggingWithin = useIsDraggingWithin( wrapper );
+
+	const [ openedWhileDragging, setOpenedWhileDragging ] = useState( false );
+	useEffect( () => {
+		if ( ! isDragging ) {
+			setOpenedWhileDragging( false );
+			return;
+		}
+
+		let timeout;
+		if ( isDraggingWithin && ! isOpen ) {
+			timeout = setTimeout( () => {
+				setOpen( true );
+				setOpenedWhileDragging( true );
+			}, 600 );
+		} else if ( ! isDraggingWithin && isOpen && openedWhileDragging ) {
+			timeout = setTimeout( () => {
+				setOpen( false );
+			}, 100 );
```

Comment:
Completely arbitrary timeouts

Reply:
I'm personally finding the delays are unhelpful. The problem is that there's no indication anything is about to happen when I first drag over, and also the way the page scrolls while dragging means it's quite easy to zoom past an area.

The sidebars for the old widget page open immediately, so I think we can replicate that for now. I took the liberty of pushing a commit to do that.